packages/driver/src/cypress.ts

```diff
@@ -209,12 +209,8 @@ class $Cypress {
   // or parsed. we have not received any custom commands
   // at this point
   onSpecWindow (specWindow, scripts) {
-    const logFn = (...args) => {
-      return this.log.apply(this, args)
-    }
-
     // create cy and expose globally
-    this.cy = $Cy.create(specWindow, this, this.Cookies, this.state, this.config, logFn)
+    this.cy = new $Cy(specWindow, this, this.Cookies, this.state, this.config)
```

Comment:
For curious reviewers, logFn isn't used by the constructor (or even the create function at this point) so it's safe to omit.